pkg/build/apk.go

```diff
@@ -111,7 +111,12 @@ func (bc *Context) InitApkKeyring() (err error) {
 			// Normalize the element as a URI, so that local paths
 			// are translated into file:// URLs, allowing them to be parsed
 			// into a url.URL{}.
-			asURI := uri.New(element)
+			var asURI uri.URI
+			if ok := strings.HasPrefix(element, "https://"); ok {
```

Comment:
```suggestion
			if strings.HasPrefix(element, "https://") {
```

(the `ok` doesn't really add much)